### PR TITLE
LPS-70667

### PR DESCRIPTION
**Pull request by ricardocousolr**
Hi @natocesarrego. This fix is for a customer. Discussion for background in LPP-24464. Tell me if you have any questions. Thanks!

### CODE REVIEW

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by natocesarrego** [comment]
ci:retest

File: modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/calendar_container.js [hunk 1]
```diff
@@ -223,7 +223,12 @@ AUI.add(
 											if (calendarResourceId == instance.get('groupCalendarResourceId') && item.get('permissions').MANAGE_BOOKINGS) {
 												defaultCalendar = item;
 											}
-											else if (calendarResourceId == instance.get('userCalendarResourceId') && defaultCalendar == null) {
+
```
Comment (natocesarrego) [inline modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/calendar_container.js]: Hi @ricardocousolr,

There is any specific reason for not use only one `if` to check if the `defaultCalendar == null`? I imagine you desire to keep the same code pattern used in the method `CalendarDisplayContext.getDefaulCalendar`, am I right?

**Comment by natocesarrego** [comment]
Hey @ricardocousolr,

One more thing, are you expecting different pop up contents independent of the Display User Events value? Please check what I've gotten:

Display User Events checked

display user events checked

Display User Events unchecked

display user events unchecked

Just for you know, I've executed these tests with a new user which is member of site Liferay.

Best.

File: modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/calendar_container.js (hunk 1 above)
Comment (ricardocousolr) [inline modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/calendar_container.js]: Hi @natocesarrego, as you imagined I tried to follow the same logic as that of `CalendarDisplayContext.getDefaultCalendar`. Maybe this type of redundant code could be avoided in the future.

**Comment by ricardocousolr** [comment]
Hi @natocesarrego. 

I've tried reproducing the screenshots you attached but I get the same result in both cases (the same as the first screenshot), and this is the intended behavior. Can you check that in the second case the site member has permission to view event details for the site's calendar? Maybe it's not checked. Let me know if the same behavior persists. 

Best.

**Comment by natocesarrego** [comment]
Hi @ricardocousolr,

I think it's checked by default, but I'll repeat the steps and see what I get, ok? 😃

Regards.

**Comment by natocesarrego** [comment]
Pull request submitted to https://github.com/brandizzi/liferay-portal/pull/396. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by natocesarrego** [comment]
Hey @ricardocousolr,

I've just rebased my code and it has worked as you said. 😄 

Thanks for your contribution!

**Comment by ricardocousolr** [comment]
I'm glad it worked, thank you!

**Pull request closed without merging**